src/format.rs

```diff
@@ -1022,6 +1022,10 @@ impl Documentable for &UntypedClauseGuard {
 
             ClauseGuard::Float { value, .. } => value.to_string().to_doc(),
 
+            ClauseGuard::Tuple { elems, .. } => "tuple"
```

Comment:
Could we have a test for formatting of Gleam code that uses tuples in guards please?

Reply:
Fixed!